src/tests/ftest/scrubber/aggregation.yaml

```diff
@@ -39,20 +39,24 @@ pool:
 container:
   type: POSIX
   control_method: daos
-  oclass: RP_2GX
   properties: "cksum:sha512,rd_fac:1"
-ior:
+ior: &ior_base
   ior_timeout: 60
-  client_processes:
-    np: 6
   test_file: daos:testFile
   dfs_destroy: false
   flags: "-v -W -w -r -R -k"
   api: DFS
+  dfs_oclass: RP_2GX
+ior_small_block_size:
+  <<: *ior_base
+  transfer_size: 50M
```

Comment:
Why 50M? This seems abnormally high

Reply:
Reverted to 5M. I am seeing random corruption behaviors on different cluster... Presently, trying to increase fault injection interval value and see how things go under CI on different clusters.